formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Encoding.kt

```diff
@@ -467,3 +481,74 @@ private fun Iterable<ByteArray>.flatten(): ByteArray {
 private fun SerialDescriptor.isByteString(index: Int): Boolean =
     getElementDescriptor(index) == ByteArraySerializer().descriptor &&
         getElementAnnotations(index).find { it is ByteString } != null
+
+private fun halfToDouble(bits: Short): Long {
```

Comment:
It also would be nice to drop a link to the specification here

Reply:
Specification of what? IEEE-754?

Reply:
To corresponding appendix of CBOR rfc, I presume